src/platform/src/common/features/create-organization/components/CreateOrganizationForm.jsx

```
@@ -48,149 +37,154 @@
+    });
   };
 
   const handleSuggestionClick = (suggestion) => {
     setFormData((prev) => ({
       ...prev,
       organizationSlug: suggestion,
     }));
+    setErrors((prev) => {
+      const c = { ...prev };
+      delete c.organizationSlug;
+      return c;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)